packages/nimble-components/src/utilities/models/string-normalizer.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Generic normalize method that removes the accents and special characters.
+ * This method can be used to normalize the string before performing diacritic-insensitive string comparisons
+ */
+export const normalizeString = (value: string): string => {
```

Comment:
lets call the file string-normalizers.ts and name this function diacriticInsensitiveStringNormalizer

Reply:
Updated the file name and method name as per the suggestion